spec/autocomplete-manager-async-spec.coffee

```diff
@@ -4,6 +4,9 @@ describe 'Async providers', ->
 
   beforeEach ->
     runs ->
+      jasmine.unspy(window, 'setTimeout')
+      jasmine.unspy(window, 'clearTimeout')
```

Comment:
I definitely want to get away from the async activation to simplify things. But why doesnt `advanceClock` work? Also, in the event that we absolutely need to use the real clock, atom has a [method](https://github.com/atom/atom/blob/964809373bbf6e621a1b307c4fae372b8fe0e525/spec/spec-helper.coffee#L219) for that.